backend/services/users/src/routes.py

```diff
@@ -0,0 +1,46 @@
+from src import app
+from flask import request, jsonify
+from src import db
+
+
+@app.route("/")
+def index():
+    return jsonify(
+        status=True,
+        message="Hello Users of " + db.name
+    )
+
+
+@app.route("/users")
+def users():
+    print(db.values)
+    userList = db.users.find()
+    
+    user = {}
+    data = []
+
+    for usr in userList:
+        user = {
+            'name' : usr['name'],
+            'privateKey' : usr['privateKey']
+        }
+        data.append(user)
+
+    return jsonify(
+        status=True,
+        data=data
+    )
+
+@app.route('/users', methods=['POST'])
+def createUser():
```

Comment:
better endpoint name: /create (/users will already be in the path)

Reply:
Yeahh, th sames were placeholders, they def could use some better names lol